### PR TITLE
[CI] Turn on basic correctness tests for V1

### PR DESCRIPTION
**Pull request by tlrmchlsmth**
Ran into some OOM issues in #9856. Turning on this test to see if the problems happen without the TP changes.

### CODE REVIEW

**Comment by WoosukKwon** [comment]
@tlrmchlsmth do we still have any issues on merging this PR?

**Comment by tlrmchlsmth** [comment]
@WoosukKwon I tried it again after merging #9856 but it's still red. Maybe there are issues with the UniprocExecutor that can be resolved one way or another :scissors:

**Comment by wubai** [comment]
https://github.com/vllm-project/vllm/blob/c2d1b075ba88271ccc23b981c223a0617afc6bfc/vllm/v1/worker/gpu_model_runner.py#L704
@tlrmchlsmth  `logits = self.model.compute_logits(hidden_states, None)` comsume much more memory than V0,  that lead to the OOM.  But I don't know how to figure out it.

**Comment by tlrmchlsmth** [comment]
>@tlrmchlsmth logits = self.model.compute_logits(hidden_states, None) comsume much more memory than V0, that lead to the OOM. But I don't know how to figure it out.

Hey, I'm pretty sure the failure was due to the engine cleaning up improperly previously, but curious what you were seeing before?

**Review by mgoin — APPROVED** [review]
Let's get this enabled
